Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -6945,7 +6945,7 @@ sub GenerateCallbackImplementationOperationBody
     push(@$contentRef, "    if (returnedException) {\n");
     if ($codeGenerator->IsPromiseType($operation->type)) {
         push(@$contentRef, "        auto* jsPromise = JSC::JSPromise::create(vm, globalObject.promiseStructure());\n");
-        push(@$contentRef, "        jsPromise->reject(&globalObject, returnedException->value());\n");
+        push(@$contentRef, "        jsPromise->rejectAsHandled(&globalObject, returnedException->value());\n");
```

Comment:
No new test is passing? We need test coverage.

Reply:
Ah, sorry, I forgot to add that.

Reply:
The tests are marked ass [Failure Pass] so not visible in the result, but this test passes. https://results.webkit.org/?suite=layout-tests&test=imported%2Fw3c%2Fweb-platform-tests%2Fnavigation-api%2Fnavigate-event%2Fintercept-handler-throws.html

Reply:
Could we now remove the [ Failure Pass ] expectation since the test is passing now?